a.js

```diff
@@ -0,0 +1,3 @@
+var bod_neme;
+var good_name;
+var bod_vor_nome;
```

Comment:
format: typos:
Possible typos in `bod_vor_nome`, suggestions:
Candidate #1 with a confidence 33%
```suggestion
var body_var_node;
```
Candidate #2 with a confidence 33%
```suggestion
var body_for_node;
```
Candidate #3 with a confidence 32%
```suggestion
var body_or_node;
```